srcpkgs/boringtun/template

```diff
@@ -0,0 +1,11 @@
+# Template file for 'boringtun'
+pkgname=boringtun
+version=vTEST2
+revision=1
+build_style=cargo
+short_desc="BoringTun is an implementation of the WireGuard® protocol designed for portability and speed."
+maintainer="TBC"
+license="BSD-3-Clause" 
```

Comment:
You have to install the license in post_install with vlicense here due to it being BSD

Reply:
Fixed, thank-you!